docs/en-us/dev/user_doc/guide/task/map-reduce.md

```diff
@@ -1,73 +1,73 @@
-- **Descriptive information**: describe the function of the node.
-- **Task priority**: When the number of worker threads is insufficient, they are executed in order from high to low, and when the priority is the same, they are executed according to the first-in first-out principle.
-- **Worker grouping**: Tasks are assigned to the machines of the worker group to execute. If Default is selected, a worker machine will be randomly selected for execution.
-- **Environment Name**: Configure the environment name in which to run the script.
-- **Number of failed retry attempts**: The number of times the task failed to be resubmitted.
-- **Failed retry interval**: The time, in cents, interval for resubmitting the task after a failed task.
-- **Delayed execution time**: the time, in cents, that a task is delayed in execution.
-- **Timeout alarm**: Check the timeout alarm and timeout failure. When the task exceeds the "timeout period", an alarm email will be sent and the task execution will fail.
-- **Resource**: Refers to the list of resource files that need to be called in the script, and the files uploaded or created by the resource center-file management.
-- **Custom parameters**: It is a user-defined parameter that is part of MapReduce, which will replace the content with ${variable} in the script.
-- **Predecessor task**: Selecting a predecessor task for the current task will set the selected predecessor task as upstream of the current task.
-
-### JAVA/SCALA Program
-
-- **Program type**: select JAVA/SCALA program.
-- **The class of the main function**: is the full path of the Main Class, the entry point of the MapReduce program.
-- **Main jar package**: is the MapReduce jar package.
+-    **Node name**: The node name in a workflow definition is unique.
+-    **Run flag**: Identifies whether this node schedules normally, if it does not need to execute, select the `prohibition execution`.
+-    **Descriptive information**: Describe the function of the node.
+-    **Task priority**: When the number of worker threads is insufficient, execute in the order of priority from high to low, and tasks with the same priority will execute in a first-in first-out order.
+-    **Worker grouping**:  Assign tasks to the machines of the worker group to execute. If `Default` is selected, randomly select a worker machine for execution.
+-    **Environment Name**: Configure the environment name in which run the script.
+-    **Times of failed retry attempts**: The number of times the task failed to resubmit.
+-    **Failed retry interval**: The time interval (unit minute) for resubmitting the task after a failed task.
+-    **Delayed execution time**: The time (unit minute) that a task delays in execution.
+-    **Timeout alarm**: Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail.
```

Comment:
Should we remove the unnessary space here?
```suggestion
- **Timeout alarm**: Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail.
```

Reply:
done changes